problem_builder/tests/integration/test_step_builder.py

```diff
@@ -1370,10 +1400,12 @@ def test_no_review_step(self):
         self.freeform_answer(None, step_builder, controls, 'This is the answer', CORRECT)
 
     def provide_freeform_answer(self, step_number, question_number, step_builder, text_input):
+        # Note: step_number is unused, but left here to be consistent the rest of methods that fill questions
         steps = step_builder.find_elements_by_css_selector('div[data-block-type="sb-step"]')
-        current_step = steps[step_number-1]
+        # There is always only a single step in the Step Builder --- the current one
+        current_step = steps[0]
         freeform_questions = current_step.find_elements_by_css_selector('div[data-block-type="pb-answer"]')
-        current_question = freeform_questions[question_number-1]
+        current_question = freeform_questions[question_number - 1]
```

Comment:
@jbzdak Nit: No need to change this line (I think we usually don't put spaces around operators in function calls or when slicing).